automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/runtimes/RuntimePatchSpec.scala

```diff
@@ -70,39 +77,132 @@ class RuntimePatchSpec extends FixtureAnyFreeSpec with LeonardoTestUtils with Le
           stoppingDoneCheckable
         ).compile.lastOrError
         _ = monitorStoppingResult.status shouldBe ClusterStatus.Starting
-        monitringStartingResult <- testTimer.sleep(50 seconds) >> streamFUntilDone(ioa, 30, 10 seconds)(
+        monitoringStartingResult <- testTimer.sleep(50 seconds) >> streamFUntilDone(ioa, 30, 10 seconds)(
           testTimer,
           startingDoneCheckable
         ).compile.lastOrError
+        clusterCopy = ClusterCopy.fromGetRuntimeResponseCopy(getRuntimeResult)
+        _ <- IO(
+          withWebDriver { implicit driver =>
+            withNewNotebook(clusterCopy, Python3) { notebookPage =>
+              //all other packages cannot be tested for their versions in this manner
+              //warnings are ignored because they are benign warnings that show up for python2 because of compilation against an older numpy
+              val res = notebookPage
+                .executeCell(
+                  "! df -H |grep sdb"
+                )
+                .get
+              res should include("22G") //disk output is always a few more gb than what's specified
```

Comment:
Really.. we specifiy `DiskSize(20)` and get `22G` from `df -H`?

Reply:
yea...not sure why